nodewatcher/modules/monitor/http/survey/management/commands/export_survey_data.py

```diff
@@ -136,8 +136,8 @@ def all_nodes_survey_graph(at):
 
     exported_graph = {
         'graph': {
-            'vertices': vertices,
-            'edges': edges,
+            'v': vertices,
+            'e': edges,
```

Comment:
Yes but we're otherwise breaking an abstraction barrier.

The rogue detection algorithm should not need to know if it's processing data that comes from just one stream (in which case the keys are 'v' and 'e') or if the data came from this function that combines data from multiple streams (in which case the keys are 'vertices' and 'edges').

So it should always rely on the same key, hence we should revert to 'v' and 'e'. This doesn't matter for the other two fields (known_nodes and timestamp)..